pafy/g.py

```diff
@@ -15,7 +15,9 @@
     'thumb': "http://i.ytimg.com/vi/%s/default.jpg",
     'bigthumb': "http://i.ytimg.com/vi/%s/mqdefault.jpg",
     'bigthumbhd': "http://i.ytimg.com/vi/%s/hqdefault.jpg",
-
+    # "only" gives you the first 100 so not in use right now
```

Comment:
That limitation probably doe't exist if you use the gdata api: https://developers.google.com/youtube/v3/docs/

Reply:
yeah added that in the beginning when reading the source and playing around with youtube (should have removed the url anyway)

Reply:
I could remove this completly since I don't use the url and added it for future reference only.
Anything else I should add, reformat or change?
I would really like the channel support in pafy and am open to criticism